### PR TITLE
feature/GAP-2090

### PR DESCRIPTION
**Pull request by john-tco**
## Description

Ticket # and link
https://technologyprogramme.atlassian.net/jira/software/projects/GAP/boards/216?selectedIssue=GAP-2090

Summary of the changes and the related issue. List any dependencies that are required for this change:

Adds migration banners for find and apply

## Type of change

Please check the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes:

- [x] Unit Test

- [ ] Integration Test (if applicable)

- [ ] End to End Test (if applicable)

## Screenshots (if appropriate):

Please attach screenshots of the change if it is a UI change:

# Checklist:

- [ ] If I have listed dependencies above, I have ensured that they are present in the target branch.
- [ ] I have performed a self-review of my code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation where applicable.
- [ ] I have ran cypress tests and they all pass locally.

### CODE REVIEW

File: src/components/notification-banner/migrationContentMap.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { LOGIN_NOTICE_TYPES } from '../../utils';
+
+export const MIGRATION_CONTENT_MAP = {
```
Comment (jgunnCO) [inline src/components/notification-banner/migrationContentMap.ts:3]: nice

File: src/utils/jwt.ts [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
-export const getJwtFromCookies = (req: NextRequest | NextApiRequest) => {
+export const getJwtFromCookies = (
+  req: NextRequest | NextApiRequest | GetServerSidePropsContext['req'],
+) => {
   const cookieValue =
     req instanceof NextRequest
-      ? req.cookies.get(USER_TOKEN_NAME)
-      : req.cookies[USER_TOKEN_NAME];
+      ? req.cookies.get(USER_TOKEN_NAME as string)
+      : req.cookies[USER_TOKEN_NAME as string];
```
Comment (jgunnCO) [inline src/utils/jwt.ts:16]: may as well assign the type on the declaration on line 7

Comment (john-tco) [reply src/utils/jwt.ts:16]: true

File: pages/notifications/manage-notifications/index.tsx [hunk 3]
```diff
@@ -118,9 +122,20 @@ export const getServerSideProps = async (ctx) => {
         },
       );
     }
+    const {
+      applyMigrationStatus = null,
+      findMigrationStatus = null,
+      migrationType = null,
+    } = ctx.query as NodeJS.Dict<string>;
```
[the remark this answers is not in the document]
Comment (john-tco) [reply pages/notifications/manage-notifications/index.tsx:129]: yeah fair lol

**Review by jgunnCO — APPROVED** [review]

**Pull request merged**